fnf/data/mag_orm.py

```diff
@@ -148,10 +148,21 @@ class AffiliationLocation(Base):
 
 if __name__ == "__main__":
     import os
+    import logging
     from dotenv import load_dotenv, find_dotenv
-    from sqlalchemy import create_engine
+    from sqlalchemy import create_engine, exc
 
     load_dotenv(find_dotenv())
+
+    # Try to create the database if it doesn't already exist.
+    try:
+        engine = create_engine(os.getenv("test_postgresdb")) 
+        conn = engine.connect()
+        conn.execute("commit") 
+        conn.execute("create database disinfo") 
+        conn.close() 
+    except exc.DBAPIError as e:
```

Comment:
This captures both a failure to create the database and the scenario where an error is raised because it already exists.
It would be best to do something like 
```python
...
except exc.DBAPIError as e:
  if isinstance(e.orig, psycopg2.errors.DuplicateDataBase):
    logging.info(e)  # this condition is fine
  else:
    logging.error(e)  # if we can't create the database we should log the error and raise an exception
    raise
```